PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+# Situation
+<!-- Describe the background or context leading to this change. Include why this work is needed (e.g., a specific problem, user need, or opportunity). What is the current state or issue that prompted this PR? -->
+
+# Task
+<!-- Explain the goal or objective of this change. What specifically needs to be achieved to resolve the situation? Clearly define the scope of the task at hand. -->
+
+# Action
+<!-- Summarize the key steps or decisions taken to accomplish the task. Include what changes were made in the codebase, architecture, or process. What actions were implemented to address the task? -->
+
+# Results
+<!-- Detail the outcomes of the actions. What improvements or changes have been achieved? Include performance gains, bug fixes, or other tangible outcomes. How will you measure or verify success? -->
+
```

Comment:
Should we add a "related issues" section?

# Related Issues
Mention any related issue numbers, tickets, or tasks this PR addresses.

Reply:
I've updated the notes section. Please let me know your thoughts.

Reply:
Yup, that works too, thanks!